Exec/science/flame_wave/initial_model.H

```diff
@@ -11,17 +11,17 @@
 //  The temperature profile is:
 //
 //         ^
-//         |
-//  T_hi   +            /\
-//         |           /  \
-//         |          /  . \
-//  T_star +---------+      \
-//         |         .   .   \
-//         |                  \
-//         |         .   .     \
-//  T_lo   +                    +-----------
-//         |         .   .
-//         +---------+---+---------------> r
+//         |             ^                                     .
```

Comment:
Well now your T_hi doesn't match the peak :-D

Reply:
fixed